cmd/testbot/main.go

```diff
@@ -35,10 +35,12 @@ var (
 type pullRequest struct {
 	Action string
 	PR     struct {
-		Head struct {
+		Number string
+		Head   struct {
 			Ref string
 			Sha string
```

Comment:
(nit: out of scope for this PR, but this should be named `SHA` rather than `Sha`)